Source/Engines/ModuleEngineConfigs.cs

```diff
@@ -129,6 +129,7 @@ public class ModuleEngineConfigs : PartModule, IPartCostModifier, IPartMassModif
         public bool literalZeroIgnitions = false; /* Normally, ignitions = 0 means unlimited.  Setting this changes it to really mean zero */
 
         public List<ConfigNode> configs;
+        internal List<ConfigNode> _filteredDisplayConfigs;
```

Comment:
Don't think MEC uses underscores in idents.

Reply:
Fixed